sources/packages/backend/apps/workers/src/workers.module.ts

```diff
@@ -65,6 +67,8 @@ import { TerminusModule } from "@nestjs/terminus";
     ApplicationService,
     ApplicationExceptionService,
     SupportingUserService,
+    SFASApplicationService,
+    SFASPartTimeApplicationsService,
```

Comment:
These should be exported from the `SFASIntegrationModule` and the `SFASIntegrationModule` should be imported into workers.
Apparently, there is a problem where we have some duplicated providers (e.g. `SFASApplicationService`). Please let's try to resolve it in an upcoming PR in this ticket.